.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 web-ext-artifacts
 addon.xpi
-/addon/build
+addon/build
+addon/manifest.json
```

Comment:
Added to .gitignore since now it's automatically generated during `npm run build` (and indirectly via `npm run package`).